src/Sylius/Bundle/UserBundle/Form/Type/UserRegistrationType.php

```diff
@@ -37,6 +38,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => $this->dataClass,
+            'validation_groups' => $this->validationGroups,
```

Comment:
What do you need to override here exactly? Not perfect, but would be better to call parent::configureOptions, otherwise we do not even know what you change unless we check parent method. Added cascade_validation? Why you need it?